cdap-api/src/main/java/co/cask/cdap/api/mapreduce/MapReduceContext.java

```diff
@@ -55,9 +67,33 @@
   void setInput(String datasetName, List<Split> splits);
 
   /**
+   * Overrides the input configuration of this MapReduce job to write to the specified dataset instance.
+   * Currently, the dataset passed in must either be an {@link InputFormatProvider} or a {@link BatchReadable}.
```

Comment:
Insert a blank line and then enclose the remaining text in paragraph tags \<p> and \</p>